### PR TITLE
Fixed bad formatted string. Fixes #1334

### LINKED ISSUE #1334: Error uploading image
Traceback (most recent call last):
  File "/home/vagrant/webproject/aroucageopark/virtualenv/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/vagrant/webproject/aroucageopark/virtualenv/local/lib/python2.7/site-packages/django/views/decorators/cache.py", line 38, in _cache_controlled
    response = viewfunc(request, *args, **kw)
  File "/home/vagrant/webproject/aroucageopark/virtualenv/local/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/vagrant/webproject/aroucageopark/virtualenv/local/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/vagrant/webproject/aroucageopark/virtualenv/local/lib/python2.7/site-packages/wagtail/wagtailimages/views/chooser.py", line 124, in chooser_upload
    if form.is_valid():
  File "/home/vagrant/webproject/aroucageopark/virtualenv/local/lib/python2.7/site-packages/django/forms/forms.py", line 162, in is_valid
    return self.is_bound and not bool(self.errors)
  File "/home/vagrant/webproject/aroucageopark/virtualenv/local/lib/python2.7/site-packages/django/forms/forms.py", line 154, in errors
    self.full_clean()
  File "/home/vagrant/webproject/aroucageopark/virtualenv/local/lib/python2.7/site-packages/django/forms/forms.py", line 353, in full_clean
    self._clean_fields()
  File "/home/vagrant/webproject/aroucageopark/virtualenv/local/lib/python2.7/site-packages/django/forms/forms.py", line 366, in _clean_fields
    value = field.clean(value, initial)
  File "/home/vagrant/webproject/aroucageopark/virtualenv/local/lib/python2.7/site-packages/django/forms/fields.py", line 606, in clean
    return super(FileField, self).clean(data)
  File "/home/vagrant/webproject/aroucageopark/virtualenv/local/lib/python2.7/site-packages/django/forms/fields.py", line 150, in clean
    value = self.to_python(value)
  File "/home/vagrant/webproject/aroucageopark/virtualenv/local/lib/python2.7/site-packages/wagtail/wagtailimages/fields.py", line 127, in to_python
    self.check_image_file_size(f)
  File "/home/vagrant/webproject/aroucageopark/virtualenv/local/lib/python2.7/site-packages/wagtail/wagtailimages/fields.py", line 120, in check_image_file_size
    filesizeformat(f.size),
TypeError: format requires a mapping

### PR DESCRIPTION
**Pull request by kaedroho**

### CODE REVIEW

**Comment by tomdyson** [comment]
Hi @jordij - do you have time to review this? Thanks!

**Comment by jordij** [comment]
will do during this week @tomdyson . very busy with client work atm :(

**Comment by gasman** [comment]
Thanks @jordij - some time this week will be fine.

File: wagtail/wagtailimages/tests/test_admin_views.py [hunk 1]
```diff
@@ -95,6 +97,21 @@ def test_add_no_file_selected(self):
...
+            'file': SimpleUploadedFile('test.png', get_test_image_file().file.getvalue()),
+        })
+
+        # Shouldn't redirect anywhere
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'wagtailimages/images/add.html')
+
+        # The form should have an error
+        # Note: \xa0 = non-blocking space
+        self.assertFormError(response, 'form', 'file', "This file is too big (1.9\xa0KB). Maximum filesize 1\xa0byte.")
```
Comment (gasman) [inline wagtail/wagtailimages/tests/test_admin_views.py]: Hard-coding "1.9 KB" in the test result is a bit of a liability - we can't count on that remaining accurate for every version of PIL / Willow that might be in use.

**Comment by gasman** [comment]
Uploading an oversized file on /admin/images/multiple/add/ now fails with the message "This file is too big (%s). Maximum filesize 1000 bytes." (with the literal "%s" in it). This is because this page was previously using `FILE_TOO_LARGE_ERROR` (which doesn't interpolate the current file's size) as distinct from `FILE_TOO_LARGE_KNOWN_SIZE_ERROR` (which does), but the PR here conflates the two messages into one.

**Comment by kaedroho** [comment]
Issues fixed

**Pull request merged**